Podcast/ProfileMiniHeader.swift

```diff
@@ -12,73 +12,62 @@ import UIKit
 
 class ProfileMiniHeader: UIView {
```

Comment:
why is there a profileminiheader and an internalprofileheaderview, can we combine these two into one class to standardize?